packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js

```diff
@@ -1433,6 +1433,26 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('Change winloss lossColor property', function(assert) {
+        const sparkline = this.createSparkline({
+            type: 'winloss',
+            winColor: 'yellow',
+            lossColor: 'blue',
+            firstLastColor: 'pink',
+            winlossThreshold: 4,
```

Comment:
I believe tests should be as concise as possible. We are testing the runtime change of the lossColor property. Do we need to initialize other properties, such as winLossThreshold or winColor?